src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -203,6 +203,22 @@ namespace NS
                 expectedIndentation: 4);
         }
 
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        public void FileScopedNamespace()
+        {
+            var code = @"using System;
+
+namespace NS;
+
+
+";
+            AssertSmartIndent(
+                code,
+                indentationLine: 4,
+                expectedIndentation: 0);
```

Comment:
can you also add an assert for the lines *Above* the namespace?  and can we have a test that just tests teh behavior of indentation if the semicolon is missing (i'm fine with any behavior, i just want it tested).

Reply:
Done